pkgs/development/libraries/speechd/default.nix

```diff
@@ -1,38 +1,88 @@
-{ fetchurl, lib, stdenv, intltool, libtool, pkgconfig, glib, dotconf, libsndfile
-, libao, python3Packages
-, withEspeak ? false, espeak
+{ stdenv, lib, pkgconfig, fetchurl, wrapPython
+, intltool, itstool, libtool, texinfo, autoreconfHook
+, glib, dotconf, libsndfile
+, pyxdg
+, withLibao ? true, libao
+, withPulse ? false, libpulseaudio
+, withAlsa ? false, alsaLib
+, withOss ? false
+, withFlite ? true, flite
+, withFestival ? false # TODO: , festival-freebsoft-utils
+, withEspeak-ng ? true, espeak-ng, sonic, pcaudiolib
 , withPico ? true, svox
+, withIvona ? false # TODO: , libdumbtts
+, defaultModule ? null
+, defaultLang ? "en"
 }:
 
-stdenv.mkDerivation rec {
+with lib;
+let
+  selectedDefaultModule =
+    if defaultModule != null then
+      defaultModule
+    else if withEspeak-ng then
+      "espeak-ng"
+    else if withPico then
+      "pico"
+    else if withFlite then
+      "flite"
+    else
+      throw "You need to enable at least one output module.";
+in stdenv.mkDerivation rec {
   name = "speech-dispatcher-${version}";
-  version = "0.8.5";
+  version = "0.8.8";
 
   src = fetchurl {
     url = "http://www.freebsoft.org/pub/projects/speechd/${name}.tar.gz";
-    sha256 = "18jlxnhlahyi6njc6l6576hfvmzivjjgfjyd2n7vvrvx9inphjrb";
+    sha256 = "1wvck00w9ixildaq6hlhnf6wa576y02ac96lp6932h3k1n08jaiw";
   };
 
-  buildInputs = [ intltool libtool glib dotconf libsndfile libao python3Packages.python ]
-             ++ lib.optional withEspeak espeak
-             ++ lib.optional withPico svox;
-  nativeBuildInputs = [ pkgconfig python3Packages.wrapPython ];
+  nativeBuildInputs = [ pkgconfig autoreconfHook intltool itstool texinfo wrapPython ];
 
-  hardeningDisable = [ "format" ];
+  propagatedBuildInputs = [ pyxdg ];
```

Comment:
`pyxdg` is already listed in `pythonPath` below.

Reply:
Done